ui/src/app/pages/data/data-set/new-dataset-modal.tsx

```diff
@@ -162,7 +162,7 @@ class NewDataSetModal extends React.Component<Props, State> {
         // Open notebook in a new tab and return back to the Data tab
         const notebookUrl = '/workspaces/' + workspaceNamespace + '/' + workspaceId +
```

Comment:
nit (and out of scope): It would be helpful to have a utility function to construct a notebook URL from the things that aren't const in it.